internal/cli/kraft/cloud/instance/create/create.go

```diff
@@ -191,6 +192,13 @@ func Create(ctx context.Context, opts *CreateOptions, args ...string) (*kcclient
 	}
 	if opts.Memory > 0 {
 		req.MemoryMB = ptr(int(opts.Memory))
+	} else {
+		// Set the default memory to the size of the image if not specified plus an
+		// arbitrary 10% buffer.  Only set the value if it is greater than 128 MiB.
+		mem := int(math.Round(float64(image.SizeInBytes / 1024 / 921)))
```

Comment:
This looks wrong, why are you dividing KiB by `921` and not `1024` to get MiB

I Get the 10% buffer, but you can just multiply by `1.1` after

(Go anyway transforms the divisions you a single op for efficiency) (I think)

Reply:
Moreover, remember we are still using bbuddy, which will waste 1/2 of the memory it allocates, so 10% is not enough for most cases

Reply:
Will double check which allocator we're using. Funny though, this was enough for my testing to work.

I can change it to `(n/1024/1024)*1.1` to make it more easily readable.

Reply:
But what i can do is just increase the memory size to, instead, to the nearest 2^N.

Reply:
Actually, just leave the 10% buffer, we can just pray to Simon to get the other allocator in